slsDetectorCalibration/moenchExecutables/moenchRawDataProcess.cpp

```diff
@@ -174,14 +174,19 @@ int main(int argc, char *argv[]) {
     int nped = 1000;
     
     int cf = 0;
+<<<<<<< HEAD
     int numberOfPackets=nrows/8;
```

Comment:
you would have to remove these merge lines for this to compile.. maybe you missed it.